src/olympia/users/templates/admin/users/userprofile/change_form.html

```diff
@@ -15,6 +15,8 @@
                          type="submit">{% trans "Ban" %}</button>
                  <button formaction="{% url 'admin:users_userprofile_reset_api_key' original.pk|admin_urlquote %}"
                          type="submit">{% trans "Reset API Key" %}</button>
+                 <button formaction="{% url 'admin:users_userprofile_reset_session' original.pk|admin_urlquote %}"
```

Comment:
We should have some style for these buttons, they look quite ugly compared to the others:

![Screen Shot 2020-03-19 at 09 54 36](https://user-images.githubusercontent.com/217628/77049045-aec86b00-69c7-11ea-91af-5fc6ef774abd.png)

Reply:
https://github.com/mozilla/addons-server/pull/13763/commits/2c6185cdf88016fd81890c7dfa84092c727597e9